e2e_test/background_ddl/basic.slt

```diff
@@ -85,6 +85,18 @@ select count(*) from rw_catalog.rw_ddl_progress;
 ----
 1
 
+onlyif snapshot-backfill
```

Comment:
A creating snapshot backfill cannot be gracefully dropped. This is workaround to wait for backfill finish before issuing the drop mv SQL.